cob_bringup_sim/scripts/move_object.py

```diff
@@ -92,6 +92,21 @@ def __init__(self):
         self.node_frequency = 100.0 # hz
         self.rate = rospy.Rate(self.node_frequency) # hz
 
+
+
+    def get_model_dist(self, x, y):
+        if self.options.model != None:
+            req = GetModelStateRequest()
+            req.model_name = self.options.model
+            rospy.wait_for_service('/gazebo/get_model_state')
+            client = rospy.ServiceProxy('/gazebo/get_model_state', GetModelState)
+            res = client(req)
+            model_dist = math.sqrt(math.pow(res.pose.position.x-x,2) + math.pow(res.pose.position.y-y,2))
+        else:
+            model_dist = 5;
```

Comment:
In the else case, the ModelState cannot be retrieved....I think in this case, the object.model is does not exist (?) and we do not want to stop....thus we would set the model_dist to something ```float('inf')```